src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,75 @@
+package duke.task;
+
+/**
+ * Represents a task.
+ */
+public class Task {
+    /**
+     * The description of the task.
+     */
+    private String description;
+
+    /**
+     * The completion status of the task.
+     */
+    private boolean isDone;
+
+    /**
+     * Creates a new instance of a Task object with attributes defined
+     * in the parameters.
+     *
+     * @param description Description of the task.
+     */
+    Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Creates a new instance of a Task object with attributes defined
+     * in the parameters.
+     * Overloaded constructor which specifies the completion status of the task.
+     *
+     * @param description Description of the task.
+     * @param done Completion status of the task.
+     */
+    Task(String description, boolean done) {
```

Comment:
Since you're already using "this" keyword, I think the param "done" can be changed to "isDone"?